CIMonitor/CIMonitor
# Show statuses on the dashboard

### What

Show the incoming statuses on the dashboard. For this the `Core.statusManager` needed to be expanded as well.

### ToDo

- [x] Show the statuses in a nice manner on the dashboard

### How to test

1. `gulp`
1. See that the dashboard is open (and empty)
1. Push some new statuses into the app using postman
1. See that the dashboard shows the statuses, newest first
